src/tutorial/runtime/configuration-consolidation/parameters.rst

```diff
@@ -205,11 +205,11 @@ Parameters can be either strings or integers:
       Add a parameter called ``station``:
 
       .. code-block:: diff
+         +[task parameters]
```

Comment:
```suggestion

         +[task parameters]
```
(Needs a blank line after the code-block directive; I'll commit this tweak on the fly)